src/main/php/scriptlet/HttpScriptlet.class.php

```diff
@@ -426,24 +420,4 @@ public function service(HttpScriptletRequest $request, HttpScriptletResponse $re
       );
     }
   }
-
-  /**
-   * This method is called to process any request and dispatches
-   * it to on of the do* -methods of the scriptlet. It will also
-   * call the <pre>doCreateSession()</pre> method if necessary.
-   *
-   * @return  scriptlet.HttpScriptletResponse the response object
-   * @throws  scriptlet.ScriptletException indicating fatal errors
-   */
-  public function process() {
```

Comment:
The only place it was used was `xp.scriptlet.Runner`, via `web[-main].php`; and the test suite.